src/hr/hr.module.ts

```diff
@@ -0,0 +1,14 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
+import { UsersService } from 'src/users/users.service';
+
+import { Hr } from './entities/hr.entity';
+import { HrController } from './hr.controller';
+import { HrService } from './hr.service';
+
+@Module({
+  imports: [TypeOrmModule.forFeature([Hr])],
+  controllers: [HrController],
+  providers: [HrService, UsersService],
```

Comment:
Zasada jest taka, że powinno się importować moduł `UsersModule` , a nie wpisywać w `providers` `UsersService`.  Importując moduł, dostajesz automatycznie dostęp do usług, które są dostarczane przez ten moduł

Reply:
Po co, skoro nie potrzebuje całego modułu? Zmienione.

Reply:
Taka koncepcja. Możesz o tym poczytać - na pewno ma to jakieś zastosowanie